chainweb.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version: 2.2
 
 name:         chainweb
-version:      1.0.2
+version:      1.1.0
```

Comment:
This PR makes breaking changes to config.

Reply:
Is this still needed?

Reply:
At least it should be only 1.0.3, I think.